src/Nordea/Components/NumberInput.elm

```diff
@@ -141,6 +147,10 @@ getStyles config =
     in
     [ fontSize (rem 1)
     , height (rem 3)
+    , pseudoElement "-webkit-outer-spin-button" [ property "-webkit-appearance" "none", margin (px 0) ]
+    , pseudoElement "-webkit-inner-spin-button" [ property "-webkit-appearance" "none", margin (px 0) ]
```

Comment:
Will this work? 🤔
```suggestion
    , pseudoElement "-webkit-outer-spin-button" [ display none ]
    , pseudoElement "-webkit-inner-spin-button" [ display none ]
```

If not then I suggest to change `(px 0)` -> `(rem 0)` or `zero`. Trying to avoid px where possible 😄

Reply:
Yes, worked great!